### PR TITLE
[SymbolGraph] Print where clause in full declaration fragments

### PR DESCRIPTION
**Pull request by bitjammer**
rdar://63233737

### CODE REVIEW

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

File: test/SymbolGraph/Symbols/Mixins/DeclarationFragments/Full/NominalTypes.swift [hunk 1]
```diff
@@ -0,0 +1,208 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -module-name NominalTypes -emit-module -emit-module-path %t/
+// RUN: %target-swift-symbolgraph-extract -module-name NominalTypes -I %t -pretty-print -output-dir %t
+// RUN: %FileCheck %s --input-file %t/NominalTypes.symbols.json --check-prefix=STRUCT
+// RUN: %FileCheck %s --input-file %t/NominalTypes.symbols.json --check-prefix=CLASS
+// RUN: %FileCheck %s --input-file %t/NominalTypes.symbols.json --check-prefix=ENUM
+// RUN: %FileCheck %s --input-file %t/NominalTypes.symbols.json --check-prefix=TYPEALIAS
+
+@frozen public struct S<T> where T: Sequence {}
```
Comment (franklinsch) [inline test/SymbolGraph/Symbols/Mixins/DeclarationFragments/Full/NominalTypes.swift:9]: Did you mean to use `@frozen` here? I guess it doesn't hurt, but we're not including this attribute in the declaration fragments yet (rdar://problem/62635346).

**Review by franklinsch — APPROVED** [review]
Looks great!

File: test/SymbolGraph/Symbols/Mixins/DeclarationFragments/Full/NominalTypes.swift (hunk 1 above)
Comment (bitjammer) [reply test/SymbolGraph/Symbols/Mixins/DeclarationFragments/Full/NominalTypes.swift:9]: Ah, yes, I've been working on that other radar in parallel. It doesn't really need to be in this particular test (yet).

**Comment by bitjammer** [comment]
@swift-ci Please smoke test

**Pull request merged**